README.md

```diff
@@ -472,7 +472,7 @@ If a Workload Resource Mapping defines `containers`, it **MUST NOT** define `.en
 ## Resource Type Schema
 
 ```yaml
-apiVersion: service.binding/v1alpha2
+apiVersion: service.binding/v1alpha3
```

Comment:
Do we need to sync the API version of both `ServiceBinding` and `ClusterWorkloadResourceMapping`?

Reply:
Technically no, practically I think it's more clear for users if they are